lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -145,6 +150,19 @@ def close_running_iperf_servers(self):
         )
         return ret_code == 0
 
+    def update_mtu_size(self, set_mtu=False):
+        """ Update MTU size in TRF server """
+        # Set MTU size to 1400 for ipv6
+        if set_mtu == True:
+          ret_code = self.exec_command("sudo /sbin/ifconfig eth3 mtu 1400")
+          if ret_code != 0:
+            return False
+        else :
+          ret_code = self.exec_command("sudo /sbin/ifconfig eth3 mtu 1500")
+          if ret_code != 0:
+            return False
+        return True
```

Comment:
why do you need to change MTU of eth3 device ?
which test cases fails if you dont change MTU?

Reply:
This is required for DL testcases to avoid fragmentation/reassembly at multiple  stages. Without this iperf output will be like this: 
-----------------------------------------------------------
Server listening on 7001
-----------------------------------------------------------
Accepted connection from 3001::2, port 38655
[ 30] local fdee:5:6c:1c7:5cc9:1c25:64a4:c275 port 7001 connected to 3001::2 port 46157
[ ID] Interval           Transfer     Bandwidth
[ 30]   0.00-1.00   sec  0.00 Bytes  0.00 bits/sec
[ 30]   1.00-1.04   sec  0.00 Bytes  0.00 bits/sec
- - - - - - - - - - - - - - - - - - - - - - - - -
[ ID] Interval           Transfer     Bandwidth
[ 30]   0.00-1.04   sec  0.00 Bytes  0.00 bits/sec                  sender
[ 30]   0.00-1.04   sec  0.00 Bytes  0.00 bits/sec                  receiver

Reply:
are you using IPv6 S1 tunnel?

Reply:
Yes its an ipv6 default bearer